components/mindmap/action-items/Fit.js

```diff
@@ -1,18 +1,37 @@
-import React, { useContext } from 'react'
-import { Maximize } from 'react-feather'
-import { Button } from 'reactstrap'
-import { runLayout } from '../../../utils/cyHelpers'
-import GlobalContext from '../../GlobalContext'
+import React, { useContext, useState } from "react";
+import { Maximize } from "react-feather";
+import { Button, Tooltip } from "reactstrap";
+import { runLayout } from "../../../utils/cyHelpers";
+import GlobalContext from "../../GlobalContext";
```

Comment:
Would've been easier to read if the existing code conventions had been followed (eg. single-quoted string literals, no semicolon at end of statement, etc). Deviating from the convention may not break the code, but it makes the diffs larger and thus more difficult to review.